go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/danielfm/aws-limits-exporter
+
+go 1.12
+
+require (
+	github.com/aws/aws-sdk-go v1.25.34
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/prometheus/client_golang v1.2.1
+)
```

Comment:
Is the current `Makefile` working after moving to go modules? Should we change the `Makefile` in any way to improve the development workflow now?

Reply:
forgot to include the new Makefile in the PR, updating as well.